http/src/main/scala/org/http4s/blaze/http/ClientSessionManagerImpl.scala

```diff
@@ -70,7 +70,7 @@ private final class ClientSessionManagerImpl(sessionCache: java.util.Map[Connect
             h2.closeNow() // make sure its closed
             it.remove()
 
-          case h2: Http2ClientSession if h2.quality > 0.1 =>
+          case h2: Http2ClientSession if 0.1 < h2.quality =>
```

Comment:
What prompted this flip?

Reply:
Just that I think it reads easier.

Reply:
Maybe while at it, `0.1` could also get a name and be even easier to read?

Reply:
Done.